test/dlc_tests/sanity/test_safety_report_file.py

```diff
@@ -57,6 +59,7 @@ def __post_init__(self):
 
 
 @pytest.mark.model("N/A")
+@pytest.mark.skipif(is_canary_context(), reason="Skipping test because it does not run on canary")
```

Comment:
This isn't necessary. The default case for any test is to be skipped in canary executions.

If we need a test to be executed on canaries, it needs to have:
```
@pytest.mark.canary("<some_message>")
```